package com.sistalk.network.error

object ExceptionHandler {
    fun handleException(e:Throwable):
}